src/App.tsx

```diff
@@ -1,24 +1,140 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable max-len */
+import React, { useState, useMemo, useEffect } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Header } from './component/Header/Header';
+import { Footer } from './component/Footer/Footer';
+import { TodoApp } from './component/TodoApp/TodoApp';
+import {
+  getTodos, deleteTodos, addTodos,
+} from './api/todos';
+import { NewTodo } from './types/NewTodo';
+import { TodoFilter } from './types/TodoFilter';
 
-const USER_ID = 0;
+const USER_ID = 10610;
+const timeDelay = 3000;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredBy, setFilteredBy] = useState(TodoFilter.ALL);
+  const [isError, setIsError] = useState('');
+  const [query, setQuery] = useState('');
+
+  const getAllTodos = async () => {
+    try {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch {
+      setIsError('Loading...');
+      setTimeout(() => {
+        setIsError('');
+      }, timeDelay);
+    }
+  };
+
+  useEffect(() => {
+    getAllTodos();
+  }, []);
+
+  const handleAddTodo = async () => {
+    if (!query.trim()) {
+      setIsError('Title can`t be empty');
+
+      return;
+    }
+
+    const newTodo: NewTodo = {
+      userId: USER_ID,
+      completed: false,
+      title: query,
+    };
+
+    try {
+      await addTodos(USER_ID, newTodo);
+      await getAllTodos();
```

Comment:
To meet task requirements: 
* disable the input until receiving a response from the API;
* immediately after sending a request create a todo with id: 0 and save it to the tempTodo variable in the state (NOT to the todos array);
* show an independent TodoItem after the list if tempTodo is not null;
temp TodoItem should have the loader (check the original markup);